### ISSUE TITLE
Event fieldLayout tabs won't open from dropdown menu click

### ISSUE THREAD

**Issue by anchovy** [verbb/Events#112]
### Describe the bug

We've got multiple tabs per event. Clicking to go to a tab in the dropdown menu throws a JS error `Invalid tab ID` in Tabs.js:191

<img width="1435" alt="image" src="https://user-images.githubusercontent.com/599005/191135777-ed2990ab-ea8b-4ead-a981-9f3b82184e0b.png">

NB - issue is the same for visible and non-visible tabs.

### Steps to reproduce

1. Goto an event & reduce your browser width/open browser developer tools
2. Ensure there's at least one tab hidden
3. Click on the drop-down menu in the tab ribbon
4. Click on a tab item
5. Check the 'Console' view for the error

<img width="473" alt="image" src="https://user-images.githubusercontent.com/599005/191134394-a8b908a0-d49d-47ad-8587-e23ccf012953.png">

### Craft CMS version

Craft Pro 3.7.54

### Plugin version

1.4.22

### Multi-site?

No

### Additional context

Navigating tabs in entries works as expected.

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/events` requirement in `composer.json` to:

```
"require": {
  "verbb/events": "dev-craft-3 as 1.4.22",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.23](https://github.com/verbb/events/releases/tag/1.4.23)
